testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -161,11 +162,13 @@ def test_volume(self):
 class TestGROWriter(TestCase, tempdir.TempDir):
     def setUp(self):
         self.universe = mda.Universe(GRO)
+        self.large_universe = mda.Universe(GRO_large)
```

Comment:
Do both `universe` and `large_universe` need to be created for each test of that class while none of the tests use both? Could `large_universe` be created only in the relevant test?